_content/canon/story-about-what-is-unquestionable-ja-1_anandajoti.md

```diff
@@ -0,0 +1,14 @@
+---
+title: "The Story about what is Unquestionable (Apaṇṇakajātaka, Ja 1)"
+translator: anandajoti
+external_url: "https://ancient-buddhist-texts.net/English-Texts/Jatakas/001-Unquestionable.pdf"
+drive_links:
+  - "https://drive.google.com/file/d/1cSvreQk1zICaCYGRO45-KKYOTtfOGq1v/view?usp=drive_link"
+course: jataka
+tags:
+  - sutta
+year: 2021
+pages: 25
+---
+
+An English translation by Bhikkhu Anandajoti of the first Jātaka story, along with a commentary on the text, which has not been translated until now. This Jataka is of the Buddha as a wise caravan merchant who avoided evil yakkhas and returned home safely with all of his retinue.
```

Comment:
A bit strange, but my naming convention for such files as these is `canon/ja1+cmy_anandajoti.md`